configs/openmrs/initializer_config/ampathforms/lamp_referral_form.json

```diff
@@ -0,0 +1,152 @@
+                "concept": "bda6b247-f1fa-40d8-9164-79aebe99bfaf"
+              },
+              "questions": [
+                {
+                  "type": "obs",
+                  "required": false,
+                  "id": "referredBy",
+                  "questionOptions": {
+                    "rendering": "textarea",
+                    "concept": "eb2cec8a-ee2e-4b12-9026-3d24e88bfab8"
+                  },
+                  "validators": []
+                },
+                {
+                  "type": "obs",
+                  "required": false,
+                  "label": "Centre de santé",
+                  "id": "healthCentre",
+                  "questionOptions": {
+                    "rendering": "radio",
+                    "concept": "276cccb6-0c0b-4f84-8947-298aceea5e62",
+                    "answers": [
+                      {
+                        "concept": "c5abded0-fda7-4c21-a1b4-8c7813a2ecef"
+                      },
+                      {
+                        "concept": "5e20d814-7520-48d1-a550-40f982aaeb1d"
+                      },
+                      {
+                        "concept": "27176fcf-1551-4278-a21c-e6188141b999"
+                      },
+                      {
+                        "concept": "0ec20507-a10f-44f3-b54f-b1a0135b34d2"
+                      },
+                      {
+                        "concept": "011073f8-3e3a-4039-8e20-92b148658c80"
+                      },
+                      {
+                        "concept": "927f79e3-678b-4d6e-9143-49b4fdeadcbf"
+                      },
+                      {
+                        "concept": "b35f66c7-9290-44c3-bb2a-24bfbd05213f"
+                      },
+                      {
+                        "concept": "82ca0e72-17b7-479e-98f2-203bbb73e9d9"
+                      },
+                      {
+                        "concept": "ef947ac7-203e-428c-b6bc-d0c2ee9027e7"
+                      },
+                      {
+                        "concept": "eda808c5-5ff8-4757-a176-6e7c3f0bb01b"
+                      },
+                      {
+                        "concept": "8612c60c-a31e-4f98-b169-945caebd30c6"
+                      },
+                      {
+                        "concept": "b01fe980-6859-421b-b302-534d85862bd8"
+                      },
+                      {
+                        "concept": "1cac8d76-46f1-46d3-a595-d29ee04cc6c1"
+                      },
+                      {
+                        "concept": "ea22c04d-e705-49f5-9557-35a47fc7840e"
+                      },
+                      {
+                        "concept": "cc1cdf23-e3d4-44da-a041-b58a81cce72b"
+                      }
+                    ]
+                  },
+                  "validators": []
+                },
+                {
+                  "type": "obs",
+                  "required": false,
+                  "label": "Motif de la référence",
```

Comment:
"Referral" might need to be translated to "Orientation" in fact. But that's not sure yet (I know we put "Référence" for HSC, but I still think that looks weird.

Reply:
Waiting for confirmation on this.